test/functional/core/fileio_spec.lua

```diff
@@ -9,9 +9,11 @@ local nvim_prog = helpers.nvim_prog
 local request = helpers.request
 local retry = helpers.retry
 local rmdir = helpers.rmdir
+local mkdir = helpers.mkdir
 local sleep = helpers.sleep
 local read_file = helpers.read_file
 local trim = helpers.trim
+local currentdir = helpers.currentdir
```

Comment:
could just use `helpers.funcs.getcwd` instead. That gets the CWD of the Nvim test instance instead of the Lua test runner.

Reply:
:checkered_flag: